frontend/src/app/features/work-packages/openproject-work-packages.module.ts

```diff
@@ -463,6 +468,7 @@ import { RecentItemsService } from 'core-app/core/recent-items.service';
     WorkPackageSplitViewComponent,
     BackButtonComponent,
   ],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
Why is this part necessary? I'm not seeing a opce- being used inside the new templates, so I don't think this would be required.

Reply:
It was necessary back then because we had some back and forth with the autocompleter but you are right that it no longer is for the current solution so I removed it again.

Reply:
Turns out I was wrong and we still need that declaration [for the `turbo-frame` tag](https://github.com/opf/openproject/actions/runs/6416153569/job/17419335271). But it does feel bad to disable the check for all custom tags within the module if all we want is angular to ignore one. I would be interested in knowing if there is a better way.